### DISCUSSION TITLE
How to use "or()" with multiple referenced table?

### DISCUSSION THREAD

**Discussion by anggaaryas** [supabase/supabase#21195] [Questions] [answered]
B - A - C

For example, i want to select A table with field from table B or from table C that equal "foo".

From the documentation,  we can use `or(... , referencedTable: 'table')` . But, it only have 1 referenced table. How to use multiple referenced table?

**Comment by GaryAustin1** [comment, marked as answer]
.or() only works with one table.  You will need to do an rpc call or view.
